bin/propolis-server/src/lib/migrate/source.rs

```diff
@@ -61,6 +62,10 @@ struct SourceProtocol<T: AsyncRead + AsyncWrite + Unpin + Send> {
 
     /// Transport to the destination Instance.
     conn: WebSocketStream<T>,
+
+    /// Saved VM Time Data, which is read earlier in the protocol (TimeDataRead
+    /// phase) than it is sent (TimeData phase)
+    time_data: Option<propolis::vmm::time::VmTimeData>,
```

Comment:
Chatted some about why this was implemented as two different phases.  If they can be consolidated into a single phase, we probably don't need to keep this around.